packages/time/src/timeMath.js

```diff
@@ -1,9 +1,16 @@
 import { Nat } from '@endo/nat';
 import { Fail, q } from '@endo/errors';
-import { mustMatch } from '@endo/patterns';
+import { mustMatch as typelessMustMatch } from '@endo/patterns';
```

Comment:
Now I'm confused. This is unlikely to be a layering constraint. Why not use the one from @agoric/internal ?

Reply:
because `@agoric/time` currently doesn't depend on anything in `@agoric`

Reply:
Is there any reason not to change it to depend on @agoric/internal? When it doesn't create a layering problem, that's what I'd do.

Reply:
I defer to @warner. If no objections then I'll do it in a follow-up PR

Reply:
And it is thematically what @agoric/internal is for.

Reply:
I'm glad you like the tred visualization ;)

Reply:
Yes, using @agoric/store as a staging ground for moving to endo makes sense to me. Thanks.